metrics/codahale.go

```diff
@@ -104,6 +104,10 @@ func (c *CodaHale) MeasureSince(key string, start time.Time) {
 	c.measureSince(key, start)
 }
 
+func (c *CodaHale) MeasureDuration(key string, duration time.Duration) {
+	go c.updateTimer(key, duration)
```

Comment:
The goroutine @zeitlinger

Reply:
OK, removed it - the goroutine in measure since is used for `MeasureFilterResponse` (the critical proxy path), so I left it.